local-setup/jupyter/dataloader/mdms_validator.py

```
@@ -0,0 +1,678 @@
+                "msgId": "search schema",
+                "authToken": self.auth_token,
+                "userInfo": self.user_info
+            },
+            "SchemaDefCriteria": {
+                "tenantId": tenant_id,
+                "codes": [schema_code]
+            }
+        }
+
+        try:
+            response = requests.post(url, json=payload, headers={'Content-Type': 'application/json'})
```

Ruff (0.15.0)

[error] Probable use of `requests` call without timeout

(S113)